x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/context_menu/context_menu.js

```diff
@@ -109,7 +109,7 @@ export class ContextMenuUi extends PureComponent {
 
           {
             activeFollowerIndexNames.length ? (
-              <FollowerIndexPauseProvider>
+              <FollowerIndexPauseProvider onConfirm={this.closePopover}>
```

Comment:
Can we close the popover as soon as any option is selected? It looks strange for it to still be available when the user's focus should be on the modal.

![image](https://user-images.githubusercontent.com/1238659/51696217-c8d00e80-1fb9-11e9-895c-81b7c122a516.png)

Reply:
it's handy to have it open if the user clicks `Cancel`. however, I see that the z-index of the overlay does not block out the context menu in the detail panel (it does block it out for the context menu in the table, however). will investigate